### ISSUE TITLE
Parameters for fine-tuning

### ISSUE THREAD

**Issue by oateiva** [cameronosmith/FlowCam#10]
Hello,

Thanks for sharing your amazing work!

In your paper, you mention that you "fine-tune the weights of our convolutional inference backbone and MLP renderer". From what I've seen in the code: 
- MLP Renderer seems to correspond to `self.renderer` in FlowCam. The model defining it is `class MLPImplicit(nn.Module)`
- CNN backbone is not directly addressed as such in the paper, but the code mentions a ResNet backbone in the `class PixelNeRFEncoder`. In Flowcam, this corresponds to `self.nerf_enc_flow`

Is that correct? Did you freeze the entire modules or just the initial layers of each module?
In the paper, you also mention that you trained the model for 7 hours. With which hyperparameters did you achieve that? (that is, learning rate, number of epochs, etc.)

Thank you!
